dask-gateway-server/dask_gateway_server/backends/kubernetes/controller.py

```diff
@@ -1149,10 +1153,17 @@ def make_pod(self, namespace, cluster_name, config, is_worker=False):
         if extra_container_config:
             container = merge_json_objects(container, extra_container_config)
 
+        annotations = self.common_annotations
+        if extra_pod_annotations:
+            annotations = merge_json_objects(annotations, extra_pod_annotations)
```

Comment:
Annotations can only be a map, this could be:

```python
annotations = self.common_annotations.copy()
annotations.update(extra_pod_annotations)
```